CMakeLists.txt

```diff
@@ -0,0 +1,125 @@
+project (nice)
```

Comment:
Yes, the autotools work (at least on linux). On Windows the supplied sln / proj file do not work quite as well. CMake has the same project file for compilation on windows and linux and obsoletes the sln files on windows.. The project file is less complex than the autoconf / automake project and does more or less the same plattform detection and compiling, and works as well on windows. If this is interesting to be added upstream I could add support for the tests as well.